pkg/imagebuilder/content.go

```diff
@@ -14,6 +16,36 @@ import (
 )
 
 type ConfigBuilder struct {
+	pullSecret      string
+	serviceBaseURL  string
+	infraEnvID      string
+	pullSecretToken string
+}
+
+func New() *ConfigBuilder {
+	pullSecret := getEnv("PULL_SECRET", "")
+	// TODO: try setting SERVICE_BASE_URL within agent.service
+	serviceBaseURL := getEnv("SERVICE_BASE_URL", "http://127.0.0.1")
```

Comment:
`192.168.122.2:8090` or `127.0.0.1:8090`

Reply:
The above suggestion is not needed after https://github.com/openshift-agent-team/fleeting/pull/5